cmd/serve/dlna/dlna.go

```diff
@@ -97,12 +100,29 @@ func newServer(f fs.Fs, opt *dlnaflags.Options) *server {
 	if friendlyName == "" {
 		friendlyName = makeDefaultFriendlyName()
 	}
+	interfaces := make([]net.Interface, 0, len(opt.InterfaceNames))
+	for _, interfaceName := range opt.InterfaceNames {
+		var err error
+		intf, err := net.InterfaceByName(interfaceName)
+		if err != nil {
+			fs.Errorf(nil, "Failed to resolve interface name: %v", err)
+			return nil
+		}
+		if !isAppropriatelyConfigured(*intf) {
+			fs.Errorf(nil, "Interface '%s' is not appropriately configured (it should be UP, MULTICAST and MTU > 0)", interfaceName)
+			return nil
```

Comment:
So I changed the `continue` to `return nil` with a nilcheck above; OK?

Reply:
I'd probably change the function to return a `(*server, error)` and return the errors properly, so

```go
		if err != nil {
			return nil, fmt.Errorf("failed to resolve interface name: %w", err)
		}
```

That would be more go-like I think as checking `newServer` for `nil` looks a bit odd.